Resources/Prototypes/Entities/Constructible/Storage/StorageTanks/fuel_tank.yml

```
@@ -9,7 +9,7 @@
   - type: Icon
     texture: Constructible/Misc/weldtank.png
   - type: Explosive
-    devastationRange: 0
+    devastationRange: 5
```

👀

---

I thought we agreed on this.

```suggestion
    devastationRange: 0
```